packages/tools/guiEditor/src/components/propertyTab/propertyTab.scss

```diff
@@ -318,6 +319,11 @@
             font-weight: 600;
         }
 
+        .disabled {
+            color: var(--backgroundGrey);
+            background-color: var(--disabledBackground);
+        }
+

```

Comment:
maybe we can define that outside, as a "disabledGridElement" or something similar, to avoid repeating the code?

Reply:
This is not only used by the grid, but by other components too